packages/terminal/src/node/terminal-server.ts

```diff
@@ -45,6 +46,16 @@ export class TerminalServer extends BaseTerminalServer implements ITerminalServe
                 this.logger.error('Error while creating terminal', error);
                 resolve(-1);
             });
+
+            const taskTerm = this.taskTerminalFactory(options);
```

Comment:
It looks like we're creating both a `taskTerminal` and a normal terminal in this `Promise`. Is that intended, or should we create or the other, depending on some conditional?

Reply:
I added this after the builds here failed but I think it can be omitted.